packages/tools/src/dconf.coffee.md

```diff
@@ -0,0 +1,35 @@
+
+# `nikita.system.dconf`
+
+dconf is a low-level configuration system and settings management used by
+Gnome. It is a replacemet of gconf, replacing its XML based database with a
+BLOB based database.
+
+## Options
+
+*   `properties` (object)
```

Comment:
Invalid markdown syntax, preview your changes on github, u'll see there is no return at the end of the line